workspaces/adventure-pack/goodies/typescript/Number.prototype.chr/index.test.ts

```diff
@@ -0,0 +1,73 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "./index";
+
+describe("Number.prototype.chr", () => {
+  describe("convert ASCII codepoint to ASCII string", () => {
+    it.each([
+      { codepoint: 65, expected: "A" },
+      { codepoint: 10, expected: "\n" },
+      { codepoint: 61, expected: "=" },
+      { codepoint: 126, expected: "~" },
+      { codepoint: 32, expected: " " },
+      { codepoint: 48, expected: "0" },
+    ])(
+      "convert ASCII codepoint $codepoint to ASCII string $expected",
+      ({ codepoint, expected }) => {
+        expect(codepoint.chr()).toBe(expected);
+      },
+    );
+  });
+
+  describe("convert codepoint to emoji", () => {
+    it.each([
+      { codepoint: 129302, expected: "🤖" },
+      { codepoint: 129412, expected: "🦄" },
+    ])(
+      "convert codepoint $codepoint to emoji $expected",
+      ({ codepoint, expected }) => {
+        expect(codepoint.chr()).toBe(expected);
+      },
+    );
+  });
+
+  describe("convert hex codepoint to string", () => {
+    it.each([
+      { codepoint: 0x404, expected: "Є" },
+      { codepoint: 0x24, expected: "$" },
+      { codepoint: 0x1f303, expected: "🌃" },
+      { codepoint: 0x1f92a, expected: "🤪" },
+    ])(
+      "convert hex codepoint $codepoint to string $expected",
+      ({ codepoint, expected }) => {
+        expect(codepoint.chr()).toBe(expected);
+      },
+    );
+  });
```

Comment:
I'm not sure if using the extra describe blocks is needed. I'd figured that using `describe` blocks is meant to group things up, but since there's only one case for each describe block, then i don't think it's necessary having it. I'd 2x check with @miorel on what he recommends here, if these extra blocks provides more clarity.

Reply:
Resolved the other comments, but interested to here @miorel thoughts on this. I could go either way.

Reply:
So I pulled the PR and ran it both with and without the inner `describe`s. I don't have a strong preference for whether we keep them or not. I think the main use case for the descriptions is when some tests fails, we want to be clear on which test it was.

The grouping with the inner `describe`s is nice when we're running a specific test file, though. If you'd like to keep them my main ask would be to make the strings less redundant with the `it` descriptions, and also pass a "noun phrase" to the `describe`.